backend/curriculum_tracking/tests/test_command_auto_assign_reviews.py

```diff
@@ -309,3 +312,19 @@ def test_that_reviewers_returned_in_order(self, _):
             result,
             expected_result[1:-1],
         )
+
+
+class auto_assign_reviewers_based_on_reviewer_team_permission_Tests(TestCase):
+    def setUp(self):
+        setup_config()
+
+    def test_auto_assign_reviewers_based_on_reviewer_team_permission_raises_no_errors(
+        self,
+    ):
+        raises_errors = False
+        try:
+            auto_assign_reviewers_based_on_reviewer_team_permission()
```

Comment:
A simpler way to asseret that it raises no errors is:

```
def test_auto_assign_reviewers_based_on_reviewer_team_permission_raises_no_errors(
        self,
    ):
    auto_assign_reviewers_based_on_reviewer_team_permission()
```

If any errors are raised then the test will fail. This is the default behavior.